docs/user/security/api-keys/index.asciidoc

```diff
@@ -0,0 +1,85 @@
+[role="xpack"]
+[[api-keys]]
+=== API Keys
+
+An API key enables you to create secondary credentials with the same or lower 
+access rights so that you can send requests on behalf of the user.  For example, 
+you might have access to an Elasticsearch cluster and want to create a cron 
+job to extract data from it on a daily basis. You can create an API key tied 
+to your credentials and configure it with the minimum access required by the 
+script, and then put the API key credentials into the cron job.  
+
+You can also use API keys to automate ingestion of new data from 
+remote sources, without a live user interaction. 
+
+You can create API Keys in the {kib} Console and view and invalidate 
```

Comment:
Instead of starting a second sentence with `you can`, how about:

`To create, view, and invalidate APIs keys in the Kibana Console, go to **Management > Security > API Keys**.